modules/monixBio/src/weaver/monixbiocompat/package.scala

```diff
@@ -0,0 +1,6 @@
+package weaver
+
+package object monixbiocompat {
+  type IOSuite       = MutableIOSuite
```

Comment:
Suites are named after the effect type, `IO`, rather than the alias `Task`.

This is consistent with how the ZIO suites were named.

Reply:
cats-effect is moving towards making its typeclasses parametric in the Error type. It doesn't mean there will be a bi-functor hierarchy, however we will be able to let users parameterise the suites : `IOSuite` will become `IOSuite[E]` for instance, which will make the naming less surprising.